packages/toggle/addon/components/nucleus-toggle.js

```diff
@@ -0,0 +1,108 @@
+import {
+  classNames,
+  attributeBindings,
+  classNameBindings,
+  tagName,
+  layout as templateLayout,
+} from '@ember-decorators/component';
+
+import defaultProp from '@freshworks/core/utils/default-decorator';
+
+import Component from '@ember/component';
+import { action, computed } from '@ember/object';
+import { alias } from '@ember/object/computed';
+import layout from "../templates/components/nucleus-toggle";
+
+/**
+  __Usage:__
+  [Refer component page](/docs/components/nucleus-toggle)
+  @class Nucleus Toggle
+  @namespace Components
+  @extends Ember.Component 
+  @public
+*/
+@templateLayout(layout)
+@tagName('div')
+@classNames('nucleus-toggle')
+@classNameBindings('_sizeClass')
+@attributeBindings('disabled:disabled', 'isChecked:checked')
+
+class NucleusToggle extends Component {
+  /**
+  * Class name is added based on _sizeClass
+  *
+  * @computed _sizeClass
+  * @type string
+  * @private
+  */
+  @computed('size')
+  get _sizeClass() {
+    let size = this.get('size');
+    return size ? `nucleus-toggle--${size}` : 'nucleus-toggle--medium';
+  }
+
+  /**
+  * Attribute name `isChecked` is used to set aria-checked state
+  *
+  * @field isChecked
+  * @type boolean
+  * @default false
+  * @public
+  */
+  @defaultProp
+  value = false;
+  @alias('value') isChecked;
```

Comment:
Documentation for `isChecked`? And looks like it's a private CP? If so, please prefix it with _

Reply:
Bump.